index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body> 
+ <div class="navmenu">
+    <header class="logo">My logo</header>
+    <nav class="menu">
+    <div class="button">
+        <img src="Union.png" alt="hamburger">
+    </div>
+
+    </nav>
```

Comment:
- [ ] You have implemented a good header 👍. Let's improve it by moving the `nav` into the `header` tag and removing the first div (navemenu) in line 14 to improve the structure and semantics. 
- [ ] I would recommend reducing the right margin of your header so your design matches the Figma design, check the screenshot of your design below:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/77190203/153394478-bfb6b2c9-a99c-4abb-8430-19e4df0c1c4a.png">